vowpalwabbit/cb_adf.cc

```diff
@@ -267,11 +269,13 @@ void cb_adf::learn_MTR(multi_learner& base, multi_ex& examples)
   examples[_gen_cs.mtr_example]->weight *= 1.f / clipped_p *
       ((float)_gen_cs.event_sum / (float)_gen_cs.action_sum);
 
+  std::swap(_gen_cs.mtr_ec_seq[0]->pred.a_s, _a_s_mtr_cs);
   // TODO!!! cb_labels are not getting properly restored (empty costs are dropped)
```

Comment:
Is this comment still relevant?

Reply:
I think it is because this touches predictions and not labels